requirements/dev.txt

```diff
@@ -1 +1,2 @@
 black; python_version >= '3.6'
+pytest==5.2.1
```

Comment:
There is this big debate about pinning or not.
I'm more in the team "let's not pin and use nighlty build to see what breaks"
But internet says I'm wrong and I admit it is not cool for reproducibility

I might have a look at https://dependabot.com/

We should also pin `appdirs` then.

What are you though on this?